File: src/main/java/com/assets/board/AssetsApplication.java
package com.assets.board;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

import java.util.TimeZone;

@EnableScheduling
@EnableAsync
@SpringBootApplication
public class AssetsApplication {
    public static void main(String[] args) {

        TimeZone.setDefault(TimeZone.getTimeZone("Europe/Kyiv"));
        SpringApplication.run(AssetsApplication.class, args);
        // 1. Capture the context returned by run()
//        ConfigurableApplicationContext context = SpringApplication.run(AssetsApplication.class, args);
//
//        // 2. Retrieve the bean by its class or name
//        TaxService service = context.getBean(TaxService.class);
//
//        // 3. Call the method
//        service.calculateDividendTax(ibReportFile, false);
    }
} 

File: src/main/java/com/assets/board/AssetsApplication.kt
package com.assets.board

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import java.util.*

@SpringBootApplication
open class AssetsApplication

fun main(args: Array<String>) {
    TimeZone.setDefault(TimeZone.getTimeZone("Europe/Kyiv"))
    SpringApplication.run(AssetsApplication::class.java, *args)
}